lib/sacastats_web/controllers/character_controller.ex

```diff
@@ -20,10 +21,14 @@ defmodule SacaStatsWeb.CharacterController do
         |> redirect(to: Routes.character_path(conn, :character_search))
 
       {:ok, %PS2.API.QueryResult{data: body}} ->
+        next_rank = Map.get(body, "battle_rank")
+        {next_rank, ""} = Integer.parse(Map.get(next_rank, "value"))
+        next_rank = next_rank + 1
+
         character_stuff = %{
-          "name" => name,
           "stat_page" => String.downcase(stat_template_name) <> ".html",
-          "response" => Map.get(body, "character_id")
+          "response" => body,
+          "next_rank" => next_rank
         }
 
         render(conn, "template.html", character: character_stuff)
```

Comment:
```suggestion
        render(conn, "template.html", character: character)
```
rename as suggested above